app/models.py

```diff
@@ -73,6 +73,27 @@ def decode_token(token):
 			# the token is invalid, return an error message
 			return "Please register or login"
 
+	#this will be used to test pagination
+	@staticmethod
+	def generate_fake_users(count=20):
```

Comment:
why are we generating fake users?

Reply:
if we generated fake events for testing... then they need to have the `created_by` attribute which is a user.